### ISSUE TITLE
Allow the definition of new Nter

### ISSUE THREAD

**Issue by GGasch** [K2SOHIGH/pcalf#19]
Either add an option to feed msa of new Nter domain to add to the detection.

Or. Instead of having the Nter msa pre loaded in pcalf, download them independantly in directory. And every msa of Nter you add to the directory will be considered for the detection.

**Comment by K2SOHIGH** [comment]
In my opinion, you might considere updating PCALF  "database" with an expert annotation of the new N-ter domain

**Comment by GGasch** [comment]
Right. But. Since our next users of Pcalf are not familiar with git hub, I am affraid they may struggle updating the core files.
